### ISSUE TITLE
v1.3 Windows, rtl_tcp.exe ERROR - A non-recoverable error occurred during a database lookup

### ISSUE THREAD

**Issue by TrueBlueBlooded** [rtlsdrblog/rtl-sdr-blog#24]
Win10 22H2, 2 x RTL-SDR Blog V3

rtl_tcp is not working for me with v1.3, works fine with v1.1.  Both the x86 and x64 builds give the same error.  I've tried variations of the listen on IP address including the machine's actual IP address.

```
C:\SDR\rtl-sdr-blog_v1.3\x64>rtl_tcp -d 0 -a 0.0.0.0
Found 2 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001
  1:  Realtek, RTL2838UHIDIR, SN: 00000002

Using device 0: Generic RTL2832U OEM
Found Rafael Micro R820T tuner
Enabled direct sampling mode, input 2
[R82XX] PLL not locked!
Disabled direct sampling mode
Tuned to 100000000 Hz.
local address 0.0.0.0 ERROR - A non-recoverable error occurred during a database lookup. .

C:\SDR\rtl-sdr-blog_v1.3\x64>cd ..\x86

C:\SDR\rtl-sdr-blog_v1.3\x86>rtl_tcp -d 0 -a 0.0.0.0
Found 2 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001
  1:  Realtek, RTL2838UHIDIR, SN: 00000002

Using device 0: Generic RTL2832U OEM
Found Rafael Micro R820T tuner
Enabled direct sampling mode, input 2
[R82XX] PLL not locked!
Disabled direct sampling mode
Tuned to 100000000 Hz.
local address 0.0.0.0 ERROR - A non-recoverable error occurred during a database lookup. .

C:\SDR\rtl-sdr-blog_v1.3\x86>cd ..\..\rtl-sdr-blog_v1.1\x64

C:\SDR\rtl-sdr-blog_v1.1\x64>rtl_tcp -d 0 -a 0.0.0.0
Found 2 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001
  1:  Realtek, RTL2838UHIDIR, SN: 00000002

Using device 0: Generic RTL2832U OEM
Found Rafael Micro R820T tuner
[R82XX] PLL not locked!
Tuned to 100000000 Hz.
listening...
Use the device argument 'rtl_tcp=0.0.0.0:1234' in OsmoSDR (gr-osmosdr) source
to receive samples in GRC and control rtl_tcp parameters (frequency, gain, ...).
```

**Comment by TrueBlueBlooded** [comment]
Same issue for me with v1.3.1

**Comment by rtlsdrblog** [comment]
Should be fixed in 234877f1b73f08588829b4cef8eb6d330b9096be and Release.zip.

Apparently compiling with Visual Studio instead of mingw broke it. The fix is simple, just make sure the 'hints' struct in rtl_tcp is initialized to zero. Apparently the VS compiler doesn't zero - init structs by default, whereas on mingw/Linux the gcc compiler does.
